components/SearchField.js

```diff
@@ -2,8 +2,12 @@
 
 const React = require('react');
 
-const SearchField = props => (
-  <input />
-);
+class SearchField extends React.Component {
```

Comment:
const is more accurate here as this should be a `pure/stateless` component. You only create Classes if they hold state.